apps/L1G/restaurant/restaurant-frontend/src/components/auth/update-user-profile/EditPhoneDialog.tsx

```
@@ -0,0 +1,82 @@
+      phone: phone,
+    },
+  });
+
+  const onSubmit = async (data: PhoneFormValues) => {
+    setUpdating(true);
+    try {
+      await onUpdate(data.phone);
+      setOpen(false);
+      form.reset({ phone: data.phone });
+    } catch (error) {
+    } finally {
```

ESLint

[error] Empty block statement.

(no-empty)